File: android/src/main/kotlin/com/example/scgateway_flutter_plugin/models/SmallcaseReturns.kt
package com.example.scgateway_flutter_plugin.models

import com.google.gson.annotations.SerializedName

data class SmallcaseReturns(
        @SerializedName("daily")
        val daily: Double? = null,

        @SerializedName("weekly")
        val weekly: Double? = null,

        @SerializedName("monthly")
        val monthly: Double? = null,

        @SerializedName("yearly")
        val yearly: Double? = null
)

File: example/lib/app/models/SmallcaseReturns.dart

import 'dart:ffi';

class SmallcaseReturns {

  final dynamic daily;

  final dynamic weekly;

  final dynamic monthly;

  final dynamic yearly;

  SmallcaseReturns({this.daily, this.weekly, this.monthly, this.yearly});

  factory SmallcaseReturns.fromJson(Map<String, dynamic> parsedJson) {

    return SmallcaseReturns(
      daily: parsedJson['daily'],
      weekly: parsedJson['weekly'],
      monthly: parsedJson['monthly'],
      yearly: parsedJson['yearly'],
    );

  }
}